makelinux.sh

```diff
@@ -16,6 +16,7 @@ usage()
     echo $'\t' "--pack                  Include packaging features. Sets validation parser on."
     echo $'\t' "--skip-samples          Skip building samples."
     echo $'\t' "--skip-tests            Skip building tests."
+    echo $'\t' "--extra                 Extra flags to CMake."
```

Comment:
My concern that extra ends up as a place to dump all the other new good thing in the future.

Can you make as something like -cxxstd 20 with default value of 14 and then just pass -DCMAKE_CXX_STANDARD=<value> to cmake?

Reply:
Sure, I've switched to a pattern like that.